iohub/display_utils.py

```diff
@@ -1,11 +1,47 @@
-# Utility functions for label-free microscopy data
+""" Utility functions for displaying data """
 
 from typing import Tuple
 
 import numpy as np
+from PIL.ImageColor import colormap
```

Comment:
Maybe also explicitly depend on `pillow` in `setup.cfg`?

Reply:
added dependency to current instal version and beyond >= 9.4.0. Is that good?